functions/Get-DbaDbccHelp.ps1

```diff
@@ -0,0 +1,102 @@
+function Get-DbaDbccHelp {
+    <#
+    .SYNOPSIS
+        Execution of Database Console Command DBCC HELP
+
+    .DESCRIPTION
+        Returns the results of DBCC HELP
+
+        Read more:
+            - https://docs.microsoft.com/en-us/sql/t-sql/database-console-commands/dbcc-help-transact-sql
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Windows and SQL Authentication supported. Accepts credential objects (Get-Credential)
+
+    .PARAMETER DbccStatement
```

Comment:
I think Dbcc will be implied - can you change this to just `Statement`

Reply:
Ok. Will update command and tests.

Reply:
ty!